jobs/e2e-test-cypress.groovy

```diff
@@ -72,7 +72,10 @@ Defaults to GIT_REVISION.""",
 ).apply()
 
 // We use the build name as a unique identifier for user notifications. 
-BUILD_NAME = "e2e-test #${env.BUILD_NUMBER} (${params.URL}: ${params.CYPRESS_GIT_REVISION})"
+BUILD_NAME = "build e2e-test #${env.BUILD_NUMBER} (${params.URL}: ${params.REVISION_DESCRIPTION})"
```

Comment:
Do you want to put "cypress" in here somewhere?

Reply:
Yeah, maybe something like "e2e-cypress-test"